Source/HyprMXAdapter.swift

```diff
@@ -191,3 +206,18 @@ extension HyprMXAdapter: HyprMXInitializationDelegate {
         initializationCompletion = nil
     }
 }
+
+internal extension HyprConsentStatus {
```

Comment:
Nit: no need to include `internal`.

Reply:
[removed](https://github.com/ChartBoost/chartboost-mediation-ios-adapter-hyprmx/pull/13/commits/54e4231ae0ed88aa43a4743cf4e2d9e2bbea6bba)